cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -412,27 +424,38 @@ private async Task LazyLoadProperties(incident @case)
                 await dynamicsContext.LoadPropertyAsync(flag, nameof(dfp_dmerflag.dfp_FlagId));
             }
 
+            //load case's medical conditions (KMCs)
+            await dynamicsContext.LoadPropertyAsync(@case, nameof(incident.dfp_incident_dfp_knownmedicalcondition));
```

Comment:
note to self: this will likely involve too much regression testing and require moving but time-box and verify